pom.xml

```diff
@@ -619,6 +619,9 @@
           <configuration>
             <reportsDirectory>${ovirt.surefire.reportsDirectory}</reportsDirectory>
             <argLine>--illegal-access=permit</argLine>
+            	<parallel>classes</parallel>
```

Comment:
Please fix formatting to match the line above - use 2 instead 1 spaces. Same for the line below